ajax/libs/valine/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "valine",
+  "filename": "Valine.min.js",
+  "description": "A simple comment system based on Leancloud.",
+  "version": "1.1.8",
+  "keywords": [
+    "Valine",
+    "comment",
+    "system",
+    "Leancloud",
+    "Minimalist",
+    "simple",
+    "easy-to-use",
+    "fast-and-safe",
+    "comment-system"
+  ],
+  "author": "xCss <xioveliu@gmail.com> (https://github.com/xCss)",
+  "homepage": "https://valine.js.org",
+  "license": "GPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/xcss/Valine.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/xcss/Valine.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/!(*escape*|*detect*)"
```

Comment:
Except master branch I did not find any *escape and *detect files in tag versions. Am I missed some versions or we just do not need such restriction?